Gemfile

```diff
@@ -15,3 +15,7 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+group :test do
+  gem "timecop"
+end
```

Comment:
@bf4 Ok. You're right. I'll move to gemspec file. I've found this article after your comment: http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Another options is remove `timecop` dependency and use fixed datetimes like: `Date.new(2015,6,6)`. What do you think?